lms/templates/discussion/_thread_list_template.html

```diff
@@ -1,24 +1,27 @@
 <%! from django.utils.translation import ugettext as _ %>
 <script type="text/template" id="thread-list-template">
     <div class="forum-nav-header">
-        <a href="#" class="forum-nav-browse" aria-haspopup="true">
+        <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
             ## There is no whitespace between these because the front-end JS code
             ## needs to precisely compute the available width for forum-nav-
             ## browse-current in order to do truncation of topic names.
-            <i class="icon fa fa-reorder"></i><span class="sr">${_("Discussion topics; current selection is: ")}</span><span class="forum-nav-browse-current">${_("All Discussions")}</span><span class="forum-nav-browse-drop-arrow">▾</span>
-        </a>
+            <span class="icon fa fa-reorder" aria-hidden="true"></span>
+            <span class="sr">${_("Discussion topics; currently listing: ")}</span>
+            <span class="forum-nav-browse-current">${_("All Discussions")}</span>
+            <span class="forum-nav-browse-drop-arrow" aria-hidden="true">▾</span>
+        </button>
         <form class="forum-nav-search">
             <div class="forum-nav-search-ff-position-fix">
                 <label>
-                    <span class="sr">${_("Search")}</span>
-                    <input class="forum-nav-search-input" type="text" placeholder="${_("Search all posts")}">
-                    <i class="icon fa fa-search"></i>
+                    <span class="sr">${_("Search forum posts")}</span>
```

Comment:
@clrux I think you missed syncing this text up with the placeholder text.  This is template that matters.